src/App.js

```diff
@@ -1,59 +1,30 @@
-import './App.css';
-
-// Modules:
-import { useEffect, useState } from 'react';
-import axios from 'axios';
-
-// Components:
-import TileOne from './components/TileOne';
-// import TileTwo from './components/TileTwo';
-
+// STYLES:
+import './styles/sass/App.scss';
+// COMPONENTS:
+import ApiImages from './components/ApiImages';
```

Comment:
Loved how your components flowed from the parent App.js to ApiImages to originalTiles to SingleTile